rpc/net.go

```diff
@@ -108,6 +114,10 @@ func (s *NetService) discover(req *rpctypes.DiscoverRequest) ([]peer.ID, error)
 
 // Discover discovers peers over the network that provide a certain coin up for `SearchTime` duration of time.
 func (s *NetService) Discover(_ *http.Request, req *rpctypes.DiscoverRequest, resp *rpctypes.DiscoverResponse) error {
+	if s.isBootnode {
+		return errUnsupportedForBootnode
+	}
```

Comment:
Same here. This seems like something a bootnode could handle and might help with troubleshooting.